### PR TITLE
Add tests disjoint ip blocks

### PR DESCRIPTION
**Pull request by adisos**

### CODE REVIEW

File: pkg/netpol/connlist/connlist_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	outputFile := "out.txt"
+	for _, testName := range testNames {
+		path := filepath.Join(getTestsDir(), testName)
+		expectedOutputFile := filepath.Join(path, expectedOutputFileName)
+		res, err := FromDir(path)
+		if err != nil {
+			t.Fatalf("Test %v: TestConnList err: %v", testName, err)
+		}
+		writeRes(ConnectionsListToString(res), outputFile)
+		expectedHash, expectedErr := getFileHashValue(expectedOutputFile)
```
Comment (elevran) [inline pkg/netpol/connlist/connlist_test.go:24]: replace with `filesHaveSameContent(f1, f2)` or similar?
You can also do `stringsHaveSameContent(s1, s2)` by using `ioutil.ReadAll` from a file and avoid using `writeRes` in the previous line

Comment (elevran) [reply pkg/netpol/connlist/connlist_test.go:24]: Also, you may want to consider using a "golden file" package to compare and update test results against expected/golden files.
For example, [golden](https://pkg.go.dev/gotest.tools/v3/golden) or [autogold](https://github.com/hexops/autogold)

**Review by elevran — COMMENTED** [review]
two suggestions for possible enhancements, nothing preventing a merge of the current PR

File: pkg/netpol/connlist/connlist_test.go (hunk 1 above)
Comment (adisos) [reply pkg/netpol/connlist/connlist_test.go]: changed to comparison of strings, using `os.ReadFile`.

**Review by zivnevo — APPROVED** [review]

**Pull request merged**